14/README.md

```diff
@@ -134,28 +136,34 @@ For this BEP, we analyze a round-robin connection strategy, similar to [Round-ro
 In this scenario, every request is forwarded to a different node, distributing the load to all the nodes, and allowing recovering from connection errors by forwarding the failed request to another healthy node.
 
 This strategy needs to keep track of the following values:
-- `E`, the list of BigchainDB endpoints passed at initialization time (we use the Python notation `len(L)` to get the length of the list).
-- `i`, the index of the BigchainDB node to use for a request (`0 ≤ i < len(L)`).
-- `T`, a list of timestamps. It has the same length of `L`. On initialization, all elements contain `0`. It is used to track the availability of the nodes. We will see in a second how it is used.
-- `DELAY`, in case a node is unreachable, how much milliseconds we should wait. This is used in combination with `T`.
+
+- `E`, the list of BigchainDB endpoints passed at initialization time.
+- `timeout`, user-specified timeout, by default there is no timeout.
+- `T`, a list of timestamps. It has the same length as `E`. On initialization, all elements contain the minimal possible timestamp `datetime.min`. It is used to track the availability of the nodes.
```

Comment:
`datetime.min` is Python specific. Instead of using a Python object, milliseconds since epoch can be used, and the `min` can be just `0`.